File: app/src/main/java/com/gopher/guardian/view/GP/GPProfileFragment.java
package com.gopher.guardian.view.GP;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import androidx.fragment.app.Fragment;
import com.gopher.guardian.R;

public class GPProfileFragment extends Fragment {

  public GPProfileFragment() {
    // Required empty public constructor
  }

  @Override
  public void onCreate(final Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
  }

  @Override
  public View onCreateView(
      final LayoutInflater inflater, final ViewGroup container, final Bundle savedInstanceState) {
    // Inflate the layout for this fragment
    return inflater.inflate(R.layout.fragment_g_p_profile, container, false);
  }
}

File: app/src/main/java/deakin/gopher/guardian/view/gp/GPProfileFragment.kt
package deakin.gopher.guardian.view.gp

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import deakin.gopher.guardian.R

class GPProfileFragment : Fragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?,
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_g_p_profile, container, false)
    }
}
